databaseconnect.py

```diff
@@ -150,3 +150,54 @@ def learn_question_response(H):
     db.commit()
     B = "Thank you! I have learnt this."
     return B,0
+
+def clear_table(table_name):
+    db = connection_to_database()
+    cur = db.cursor()
+
+    if table_name in ("question_table","statement_table"):
+        tables_to_be_deleted = ["question_table","statement_table"]
```

Comment:
Really minor nitpicking here. Since the list of objects from where data will be deleted will not be modified after the list has been defined, I'd suggest the use of a tuple instead of a list (read about mutable vs immutable objects). Again, really tiny issue and just thought I'd share this thought as it would make more sense and have a greater impact in other scenarios.

Reply:
I don't know how this skipped my sight. You may notice I used a tuple for the `if` condition and did certainly want to use one for the `tables_to_be_deleted`. I'll make the change.